logistics_planning_invoicing/wizards/logistics_schedule_existing_account_move_wizard_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_logistics_schedule_existing_account_move_wizard" model="ir.ui.view">
+        <field name="name">logistics.schedule.existing.account.move.wizard.tree</field>
+        <field name="model">logistics.schedule.existing.account.move.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="carrier_id"/>
+                        <field name="journal_id" required="1"/>
+                        <field name="logistics_schedule_ids" mode="tree">
+                            <tree string="">
+                                <field name="state" invisible="1"/>
+                                <field name="origin" />
+                                <field name="partner_id" />
+                                <field name="transport_type" string="Type"/>
+                                <field name="product_id" />
+                                <field name="product_uom_qty" string="Qty"/>
+                                <field name="logistics_price_unit" string="Ord. Pr."/>
+                            </tree>
+                        </field>
+                        <field name="domain_invoice_line_ids" invisible="1"/>
+                        <field name="invoice_line_id" 
```

Comment:
I've realized that it could be more usable splitting this selection in two steps:
* First, user selects the target invoice (so it's possible to search across invoice number in a very simple way). The invoice Many2one field is filtered by journal and carrier.
* Second, once invoice is selected, we enable invoice line selection, that is filtered by invoice (and exclude for tab), a simpler filter. This field will be only available to be selected when invoice is filled, and we'll keep current display name format.

With these improvements there's no longer need of custom domain field nor invoice line `name_search`.

What do you think?